lib/lita/handlers/totems.rb

```diff
@@ -267,6 +302,19 @@ def queued_by_user(user_id)
         end
       end
 
+      def send_stats_to_signalFX(metric, value)
```

Comment:
Nit: What do you think about creating a static signalFX class that can be called instead of adding the logic to this class?